aad/Dependencies.toml

```diff
@@ -0,0 +1,51 @@
+[[dependency]]
+org = "ballerina"
+name = "log"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "regex"
+version = "0.7.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "crypto"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "time"
+version = "2.0.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "xmldata"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "os"
+version = "0.8.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "io"
+version = "0.6.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "url"
+version = "1.1.0-beta.1"
+
+[[dependency]]
+org = "ballerina"
+name = "mime"
+version = "1.1.0-beta.1"
```

Comment:
Since the latest version is beta2, please change everything to `beta.2`

Reply:
Updated to beta 2. Won't we release the connector in beta 1?

Reply:
Nope. No release in beta1. We might release the connector in beta2 or sometimes we'll go to beta3 directly. ;)